backend/app/converters/lib/marcxml_base_map.rb

```diff
@@ -225,42 +223,6 @@ def corp_template
           :obj => :name_corporate_entity,
           :rel => :names,
           :map => name_corp_map,
-        },
-        "//datafield[@tag='410']" => {
-          :obj => :name_corporate_entity,
-          :rel => :names,
-          :map => name_corp_map,
-          :defaults => {
-            :name_order => 'direct',
-            :source => 'ingest'
-          }
-        },
-        "//datafield[@tag='411']" => {
-          :obj => :name_corporate_entity,
-          :rel => :names,
-          :map => name_corp_map,
-          :defaults => {
-            :name_order => 'direct',
-            :source => 'ingest'
-          }
-        },
-        "//datafield[@tag='610']" => {
-          :obj => :name_corporate_entity,
-          :rel => :names,
-          :map => name_corp_map,
-          :defaults => {
-            :name_order => 'direct',
-            :source => 'ingest'
-          }
-        },
```

Comment:
This seems to be the source of the JIRA example's duplication.  It's already mapped below with `"datafield[@tag='611']" => mix(corp_template, agent_as_subject, corp_variation),`.